### ISSUE TITLE
Executor is already running error

### ISSUE THREAD

**Issue by MichaelHirn** [rlay-project/rlay-client#49]
I used the latest redisgraph commit. Running it with the `redisgraph` config works fine, when I tried to `cargo run -- client --config rlay.config.toml` with the neo4j config the `rlay-client` did not start and crashed with the following message

```
thread '<unnamed>' panicked at 'nested block_on: EnterError { reason: "attempted to run an executor while another executor is already running" }', src/libcore/result.rs:1084:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

**Comment by hobofan** [comment]
Should have been fixed by #50. Would be great if you can check and close if possible.

**Comment by MichaelHirn** [comment]
Confirmed! Closed by #50
